vortex-io/src/dispatcher/tokio.rs

```diff
@@ -15,7 +15,7 @@ trait TokioSpawn {
 /// A [dispatcher][Dispatch] of IO operations that runs tasks on one of several
 /// Tokio `current_thread` runtimes.
 #[derive(Debug)]
-pub(super) struct TokioDispatcher {
+pub struct TokioDispatcher {
```

Comment:
The intention was that the different variants would stay private, and you would use the type erased `IoDispatcher` factory method to create the one you want. Is there a reason we need to expose this publicly?